### PR TITLE
Keep lastSuccessfulProvision annotation

### PR DESCRIPTION
**Pull request by aleixhub**

### CODE REVIEW

File: roles/agnosticv/templates/catalog_item.yml.j2 [hunk 1]
```diff
@@ -16,6 +16,9 @@ metadata:
 {% if _total_ratings_annotation != '' %}
     babylon.gpte.redhat.com/totalRatings: {{ _total_ratings_annotation | to_json }}
 {% endif %}
+{% if _last_successful_provision_annotation != '' %}
+    babylon.gpte.redhat.com/lastSuccessfulProvision: {{ _last_successful_provision_annotation | to_json }}
+{% endif %}
```
Comment (jkupferer) [inline roles/agnosticv/templates/catalog_item.yml.j2:21]: What sets `babylon.gpte.redhat.com/lastSuccessfulProvision`?

Comment (aleixhub) [reply roles/agnosticv/templates/catalog_item.yml.j2:21]: Is a timestamp that the catalog-manager operator sets by querying the reporting db

File: roles/agnosticv/tasks/generate_catalog_item_tasks.yml [hunk 2]
```diff
@@ -94,6 +94,7 @@
     _stage: "{{ stage }}"
     _rating_label: "{{ r_get_catalog_item.resources[0].metadata.labels['babylon.gpte.redhat.com/rating'] | default('') }}"
     _total_ratings_annotation: "{{ r_get_catalog_item.resources[0].metadata.annotations['babylon.gpte.redhat.com/totalRatings'] | default('') }}"
+    _last_successful_provision_annotation: "{{ r_get_catalog_item.resources[0].metadata.annotations['babylon.gpte.redhat.com/lastSuccessfulProvision'] | default('') }}"
```
Comment (jkupferer) [inline roles/agnosticv/tasks/generate_catalog_item_tasks.yml:97]: I believe this also needs updated?

Comment (aleixhub) [reply roles/agnosticv/tasks/generate_catalog_item_tasks.yml]: good catch! thanks

**Review by jkupferer — APPROVED** [review]

**Pull request merged**